tests/spec.html

```diff
@@ -5,13 +5,35 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>QUnit Tests - Week 3</title>
         <link rel="stylesheet" href="https://code.jquery.com/qunit/qunit-2.1.1.css">
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css?family=Cinzel:400,700" rel="stylesheet">
+        <title>Test - API Project</title>
     </head>
     <body>
+        <header class="container">
+            <h1>Game Of Thrones <br> Actor Lookup</h1>
+            <p>Find the actor information for the characters in the Game of Thrones books</p>
+        </header>
+        <main id="bookWrapper" class="container">
+            <ul id="bookList">
+                <li data-bookId="1"><a href="#">A Game of Thrones</a></li>
+                <li data-bookId="2"><a href="#">A Clash of Kings</a></li>
+                <li data-bookId="3"><a href="#">A Storm of Swords</a></li>
+                <li data-bookId="5"><a href="#">A Feast For Crows</a></li>
+                <li data-bookId="8"><a href="#">A Dance with Dragons</a></li>
+            </ul>
+        </main>
+        <section id="characterWrapper" class="container">
+            <ul id="characterList"></ul>
+        </section>
+
         <div id="qunit"></div>
         <div id="qunit-fixture"></div>
         <script src="https://code.jquery.com/qunit/qunit-2.1.1.js"></script>
-        <script src="../main.js"></script>
-        <script src="./fixtures/mockData.js"></script>
-        <script src="./tests.js"></script>
+        <script type="text/javascript" src="config.js"></script>
+        <script type="text/javascript" src="request.js"></script>
+        <script type="text/javascript" src="../main.js"></script>
+        <script type="text/javascript" src="./fixtures/mockData.js"></script>
+        <script type="text/javascript" src="./tests.js"></script>
```

Comment:
inconsistent stuff - src on lines 33 and 34 doesn't have ./ at the start, while line 37 does